### PR TITLE
Move Quote and CfdOrder projections inside projection actor

### PR DESCRIPTION
**Pull request by klochowicz**
This task is a major one, so I'm submitting in stages to make it easier for both reviewers and myself (hopefully less rebase conflicts)

Still need to be done:
- connected takers (will be replaced by "counterparties")
- Cfds (will incorporate all logic of CfdWithAuxData)

### CODE REVIEW

File: daemon/tests/happy_path.rs [hunk 1]
```diff
@@ -46,8 +46,8 @@ async fn taker_takes_order_and_maker_rejects() {
     taker.take_order(received.clone(), Usd::new(dec!(10))).await;
 
     let (taker_cfd, maker_cfd) = next_cfd(taker.cfd_feed(), maker.cfd_feed()).await.unwrap();
-    assert_is_same_order(&taker_cfd.order, &received);
-    assert_is_same_order(&maker_cfd.order, &received);
+    assert_eq!(order_from_cfd(&taker_cfd), received);
+    assert_eq!(order_from_cfd(&maker_cfd), received);
```
Comment (thomaseizinger) [inline daemon/tests/happy_path.rs:50]: Maybe just asserting the same ID is enough? Unit tests should / could cover the actual projection.

**Review by thomaseizinger — APPROVED** [review]

File: daemon/tests/happy_path.rs (hunk 1 above)
Comment (klochowicz) [reply daemon/tests/happy_path.rs:50]: perhaps, but I tried to just keep the tests the way they were. :) ideally, this should all disappear, we should really address the TODO and compare the Cfd on both sides.

**Comment by klochowicz** [comment]
bors retry

**Pull request merged**